tests/phpunit/tests/rest-api/wpRestTemplatesController.php

```diff
@@ -845,4 +845,45 @@ public function test_get_template_fallback() {
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertSame( 'index', $response->get_data()['slug'], 'Should fallback to `index.html` when  ignore_empty is `true`.' );
 	}
+
+	/**
+	 * @ticket 57851
+	 * @covers WP_REST_Templates_Controller::prepare_item_for_database
```

Comment:
```suggestion
	 * @ticket 57851
	 *
	 * @covers WP_REST_Templates_Controller::prepare_item_for_database
```

Reply:
Hi @mukeshpanchal27 

Thanks for your review :raised_hands: 

Should we add a blank line between `@ticket` and `@covers`? Is this defined in some documentation? Asking that because the other functions in this file don't add a blank line.